WinFormsUI/MainForm.cs

```diff
@@ -9,6 +9,7 @@ public partial class MainForm : Form
         public MainForm()
         {
             InitializeComponent();
+            StateManager.Load();
```

Comment:
Don't call it here, but in State.Init. UI code doesn't need to know about state management